### ISSUE TITLE
[Wizard] why do we still create a dependency to Xpand/Xtend(1) in the generated target file

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext-core#824]
```
<unit id="org.eclipse.xpand" version="0.0.0"/>
<unit id="org.eclipse.xtend" version="0.0.0"/>
<unit id="org.eclipse.xtend.typesystem.emf" version="0.0.0"/>
```
why is this still there, is it needed?

(see domain model example too)

**Comment by szarnekow** [comment]
> any idea?

Sounds to me like an oversight.

**Comment by cdietrich** [comment]
Fixed in 2.15
